opencti-platform/opencti-front/src/private/components/trash/all/DeleteOperationLine.tsx

```diff
@@ -0,0 +1,175 @@
+import { DeleteOperationsLinesPaginationQuery$variables } from './__generated__/DeleteOperationsLinesPaginationQuery.graphql';
+import { DeleteOperationLine_node$data } from './__generated__/DeleteOperationLine_node.graphql';
+import DeleteOperationPopover from './DeleteOperationPopover';
+import { useFormatter } from '../../../../components/i18n';
+import ItemIcon from '../../../../components/ItemIcon';
+import type { Theme } from '../../../../components/Theme';
+import { DataColumns } from '../../../../components/list_lines';
+import ItemEntityType from '../../../../components/ItemEntityType';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  item: {
+    paddingLeft: 10,
+    height: 50,
+  },
+  itemIcon: {
+    color: theme.palette.primary.main,
+  },
+  bodyItem: {
+    height: 20,
+    fontSize: 13,
+    float: 'left',
+    whiteSpace: 'nowrap',
+    overflow: 'hidden',
+    textOverflow: 'ellipsis',
+    paddingRight: 10,
+  },
+  itemIconDisabled: {
+    color: theme.palette.grey?.[700],
+  },
+}));
+
+interface DeleteOperationLineComponentProps {
+  dataColumns: DataColumns;
+  node: DeleteOperationLine_node$data;
+  paginationOptions: DeleteOperationsLinesPaginationQuery$variables;
+}
+
+const DeleteOperationLineComponent: React.FC<DeleteOperationLineComponentProps> = ({
+  dataColumns,
+  node,
+  paginationOptions,
+}) => {
+  const classes = useStyles();
+  const { fldt } = useFormatter();
+  return (
+    <ListItem
+      classes={{ root: classes.item }}
+      divider={true}
+      button={true}
+    >
+      <ListItemIcon classes={{ root: classes.itemIcon }}>
+        <ItemIcon type={node.main_entity_type} />
+      </ListItemIcon>
+      <ListItemText
+        primary={
+          <div>
+            <div
+              className={classes.bodyItem}
+              style={{ width: dataColumns.main_entity_type.width }}
+            >
+              <ItemEntityType entityType={node.main_entity_type} />
+            </div>
+            <div
+              className={classes.bodyItem}
+              style={{ width: dataColumns.main_entity_name.width }}
+            >
+              {node.main_entity_name}
+            </div>
+            <div
+              className={classes.bodyItem}
+              style={{ width: dataColumns.deletedBy.width }}
+            >
+              {node.deletedBy?.name}
+            </div>
+            <div
+              className={classes.bodyItem}
+              style={{ width: dataColumns.timestamp.width }}
+            >
+              {fldt(node.timestamp)}
+            </div>
+          </div>
+        }
+      />
+      <ListItemSecondaryAction>
+        <DeleteOperationPopover
+          mainEntityId={node.id}
+          deletedCount={node.deleted_elements.length}
+          paginationOptions={paginationOptions}
+        />
+      </ListItemSecondaryAction>
+    </ListItem>
+  );
+};
+
+export const DeleteOperationLine = createFragmentContainer(
```

Comment:
`createFragmentContainer` is an old API of relay. Is it possible to migrate to hooks? It will be nice to have a list page using the actual relay API to serve as example for others